### PR TITLE
fix-253 - Included Handover tool in API gateway

### PR DESCRIPTION
**Pull request by riginoommen**
# Resolves 253

### 

# Explain the feature/fix
Included MoD Handover service to the API Gateway.

## Does this PR introduce a breaking change
No

## Screenshots
N/A
- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [x] Does the change have appropriate unit tests?
- [x] Did tests pass?
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [x] Was this feature demo'd and the design review approved?

### CODE REVIEW

File: packages/api-gateway-service/.env.example [hunk 1]
```diff
@@ -20,13 +20,15 @@ COOKIE_PARSER=
...
 # SEARCH_SERVICE_SERVICE_HOST="localhost"
+# MOD_SERVICE_SERVICE_HOST="localhost"
 
 ## Service Ports
 # HOME_SERVICE_SERVICE_PORT="8081"
 # FEEDBACK_SERVICE_SERVICE_PORT="8082"
 # USER_SERVICE_SERVICE_PORT="8083"
 # NOTIFICATIONS_SERVICE_SERVICE_PORT="8084"
 # SEARCH_SERVICE_SERVICE_PORT="8085"
+# MOD_SERVICE_SERVICE_PORT="8085"
```
Comment (ghanlohar) [inline packages/api-gateway-service/.env.example:31]: You might want to keep the port next number i.e. 8086 or some other number.

**Review by ghanlohar — CHANGES_REQUESTED** [review]

File: packages/api-gateway-service/.env.example (hunk 1 above)
Comment (ghanlohar) [inline packages/api-gateway-service/.env.example:31]:
```suggestion
# MOD_SERVICE_SERVICE_PORT="8086"
```

**Review by ghanlohar — APPROVED** [review]
LGTM

**Review by deshmukhmayur — APPROVED** [review]
LGTM.

**Pull request merged**